### DISCUSSION TITLE
Finetuning CUDA version

### DISCUSSION THREAD

**Discussion by green-s** [erew123/alltalk_tts#53] [General]
Just wanted to let you know that I ignored the notice about using CUDA 11.8 for finetuning and proceeded with CUDA 12.1 (nvcc installed in my conda environment using `conda install cuda-nvcc cuda-nvcc-dev_win-64` and `conda env config vars set CUDA_PATH=$env:CONDA_PREFIX CUDNN_LIB=$env:CONDA_PREFIX\Library`) and it seemed to work fine. Is there a good reason you recommend 11.8?

**Comment by erew123** [comment]
yes the external scripts (not maintained by me) explicitly ask/hunt for a copy of cublas64_11.dll

![image](https://github.com/erew123/alltalk_tts/assets/35898566/6dd38626-c635-43cd-b17f-c770bfd3aacb)

In any tests where that wasn't available on a search path the finetuning would fail. 

You may want to to load your python environment that it worked in and hunt your search paths to confirm if that file does or doesnt exist in one of them. If it did work with only 12.1 installed, then they must have updated the external scripts at some point.

**Reply by green-s** [reply]
Ah yeah sorry, still had 11.8 installed globally apparently, it must've found that.

**Comment by erew123** [comment]
No probs. Mystery solved at least :)
